dspace-api/src/main/java/org/dspace/authenticate/ShibAuthentication.java

```diff
@@ -305,7 +298,10 @@ public int authenticate(Context context, String username, String password,
 			}
 
 			// Get the Shib supplied affiliation or use the default affiliation
-			List<String> affiliations = findMultipleAttributes(request, roleHeader);
+                        List<String> affiliations = new ArrayList<String>();
+                        if (context.getCurrentUser().getMetadata(roleHeader) != null) {
```

Comment:
This seems wrong. A roleHeader is a HTTP header set by the Shibboleth daemon. So using that header name to request ePerson metadata makes no sense. The original code is correct since we use that for various clients.